recipe/meta.yaml

```diff
@@ -59,23 +59,24 @@ requirements:
     - tornado<5.0
     - simplejson==3.16.0
 
+test:
+  imports:
+    - aiida
+    - aiida.cmdline
+    - aiida.common
+    - aiida.manage
+    - aiida.restapi
+    - aiida.schedulers
+    - aiida.transports
+  commands:
+    - reentry scan -r aiida
+    - verdi --help
+    - verdi --version
+#    - verdi status
```

Comment:
was this commented out because it returns exit code 1?
If yes, then just remove this commented line

Reply:
Yes, I just wanted to know if the tests pass and ask you afterwards if you know about that verdi status error. I also do not know if it is best practice to put that reentry command in there...

Reply:
> I also do not know if it is best practice to put that reentry command in there...

I think it's fine.